mitm.default.ini

```diff
@@ -0,0 +1,167 @@
+#
+# This uses the logging.config.dictConfig format and nested keys
+# (`[handlers:stderr]`) should be in separate sections due to INI
+# limitations.
+#
+# The settings below are the PyRDP defaults.
+[logs]
+# dictConfig version. Do not modify.
+version = 1
+# A partial logger name to restrict console output to.
+# wildcards are supported. This is overridden by the `-F` flag if
+# present and does not affect file logging.
+filter = pyrdp
+
+
+# -----------------------------------------------------------------
+# Loggers and Verbosity Levels
+#
+# To disable a logger, simply remove its `level = ...` line.
+# Severity can be one of DEBUG, INFO, WARNING, ERROR or CRITICAL
+# Levels can be customized per handler and logger.
+# Note that PyRDP's `-L` switch affects the root logger.
+# -----------------------------------------------------------------
+# Root level logger for PyRDP.
+[logs:loggers:pyrdp]
+handlers = console, mitm
+level = ${vars:level}
+
+# Connection Logging
+[logs:loggers:pyrdp.mitm.connections]
+handlers = connections
+level    = ${vars:level}
+
+# Crawler Logging
+[logs:loggers:crawler]
+handlers = crawl_json, crawl_txt
+level    = ${vars:level}
+
+# SSL Secret Logging
+[logs:loggers:ssl]
+handlers = ssl, ssl_console
+# Always log SSL master secrets.
+level    = DEBUG
+
+
+# -----------------------------------------------------------------
+# Handlers
+# -----------------------------------------------------------------
+# Outputs to stderr
+[logs:handlers:console]
+class     = logging.StreamHandler
+formatter = default
+stream    = ext://sys.stderr
+# filters   = []
+
+# Outputs to mitm.log
+[logs:handlers:mitm]
+class     = logging.handlers.TimedRotatingFileHandler
+filename  = ${vars:output_dir}/${vars:log_dir}/mitm.log
+when      = D
+formatter = default
+
+# Outputs connections to mitm.json
+[logs:handlers:connections]
+class     = logging.FileHandler
+filename  = ${vars:output_dir}/${vars:log_dir}/mitm.json
+formatter = json
+
+
+# Outputs to crawl.log
+[logs:handlers:crawl_txt]
+class     = logging.FileHandler
+filename  = ${vars:output_dir}/${vars:log_dir}/crawl.log
+formatter = compact
+
+# Outputs to crawl.json
+[logs:handlers:crawl_json]
+class     = logging.FileHandler
+filename  = ${vars:output_dir}/${vars:log_dir}/crawl.json
+formatter = json
+
+# Outputs SSL secrets
+[logs:handlers:ssl]
+class     = logging.FileHandler
+filename  = ${vars:output_dir}/${vars:log_dir}/ssl.log
+formatter = ssl
+
+[logs:handlers:ssl_console]
+class     = logging.StreamHandler
+stream    = ext://sys.stderr
+formatter = ssl
+
+# -----------------------------------------------------------------
+# Formatters
+# -----------------------------------------------------------------
+# Textual / Default Format
+[logs:formatters:default]
+() = pyrdp.logging.formatters.VariableFormatter
+fmt = [{asctime}] - {levelname} - {sessionID} - {name} - {message}
+style = {
+
+# Default session ID name. Do not remove.
```

Comment:
```suggestion
# Default session ID naming strategy. Do not remove.
# Right now we name sessions based on a random name and a large but memorizable digit.
# This was favored over the use of a partial hash because it is easier to remember and discuss.
```

Reply:
The comments that you suggest are confusing, because this SessionID setting is the one that is used for logs that are not tied to a specific session. It does not influence the per-session naming strategy, as far as I know. I could add some documentation to clarify that this is the "name for the global session" or something.